DEVELOPMENT.md

```diff
@@ -96,8 +96,8 @@ it.
 If you want this `tkn` binary in your `$PATH`, a couple options are:
 
 
-1. Use `go install ./cmd/` to install the binary into your `$GOBIN`. Rerun
-   `go install` when you want to update the binary:
+1. Use `go install ./cmd/...` to install the binary into your `$GOBIN`. Rerun
```

Comment:
Actually it should be `./cmd/tkn`, as `cmd/docs` is more for "internal" use only, it's not really meant to be installed in `$GOBIN` :angel:

Reply:
Thanks for pointing this out. I've gone ahead and updated all `go build` and `go install` commands to use `tkn` instead of `...`.